docs/references/quotas.md

```diff
@@ -3,7 +3,7 @@ sidebar_position: 5
 title: Tenant Quotas & Limits
 ---
 
-This page describes the rate limits for all PaaS GDN customers. Each quota applies on a per-Region basis unless otherwise specified. Contact support@macrometa.com if you would like to request higher metered tier limits or learn about [dedicated fabrics](https://www.macrometa.com/enterprise).
+This page describes the rate limits for all PaaS GDN customers. Each quota applies on a per-Region basis unless otherwise specified. [Dedicated fabrics](https://www.macrometa.com/enterprise) can handle up to 350,000 operations per second. Reach out to support to learn more -> support@macrometa.com.
```

Comment:
```suggestion
This page describes the rate limits for all PaaS GDN customers. Each quota applies on a per-Region basis unless otherwise specified. [Dedicated fabrics](https://www.macrometa.com/enterprise) can handle up to 350,000 operations per second. Reach out to support to learn more, contact support@macrometa.com.
```